src/corelibs/U2Designer/src/PropertyWidget.cpp

```diff
@@ -423,16 +423,22 @@ ComboBoxWithChecksWidget::ComboBoxWithChecksWidget(const QVariantMap &_items, QW
 QVariant ComboBoxWithChecksWidget::value() {
     QStringList sList;
     const QList<QString> &keys = items.keys();
-    foreach (const QString &key, keys) {
+    for (const QString &key: qAsConst(keys)) {
```

Comment:
'keys' are already bus values here.
The 'keys' should be converted to display names here:
QStandardItem *item = new QStandardItem(keyToDisplayNameMap[it.key()]);